crawler/build.sbt

```diff
@@ -0,0 +1,43 @@
+name := "newspaper-crawler"
+
+version := "1.0"
+
+scalaVersion := "2.12.1"
```

Comment:
Ideally the version should not matter. Who knows ... maybe we will write one service in PHP ...